DSCResources/cChocoPackageInstall/cChocoPackageInstall.psm1

```diff
@@ -405,6 +405,8 @@ Function Upgrade-Package {
 }
 
 function Get-ChocoInstalledPackage ($action) {
+    $env:ChocolateyInstall = Split-Path -parent (Get-Command choco.exe).Source

```

Comment:
This results in `C:\ProgramData\chocolatey\bin` (using the default install location). It should be `c:\programdata\chocolatey`. I'm not keen in this being overridden each time but rather testing for the empty environment variable.